### ISSUE TITLE
Allow specifing multiple access widener files to apply in a project

### ISSUE THREAD

**Issue by i509VCB** [FabricMC/fabric-loom#340]

**Comment by modmuss50** [comment]
Is this per source set? If so it would require a loader change. I don’t quite fully understand what your asking for?

**Comment by i509VCB** [comment]
Essentially I need to inherit access wideners from another subproject by applying both the access widener from the main and subproject. I do think it would require adding an array syntax to loader though for the access wideners

**Comment by modmuss50** [comment]
Yes, this would need a change to loader, and would prob quite some gradle stuff on your end to copy it from the sub project into a common place. Not sure this is too high on the todo list tbh
